install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,431 @@
+      when: openshift_ovirt_all_in_one is defined | ternary((openshift_ovirt_all_in_one | bool), false)
+----
+
+. Create an Ansible inventory file, *_/etc/ansible/openshift_3_11.hosts_*, with the following content:
++
+[source,yaml]
+----
+[workstation]
+localhost ansible_connection=local
+
+[all:vars]
+openshift_ovirt_dns_zone="{{ public_hosted_zone }}"
+openshift_web_console_install=true
+openshift_master_overwrite_named_certificates=true
+openshift_master_cluster_hostname="openshift-master.{{ public_hosted_zone }}"
+openshift_master_cluster_public_hostname="openshift-public-master.{{ public_hosted_zone }}"
+openshift_master_default_subdomain="{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+openshift_deployment_type=openshift-enterprise
+openshift_service_catalog_image_version="{{ openshift_image_tag }}"
+
+[OSEv3:vars]
+# General variables
+debug_level=1
+containerized=false
+ansible_ssh_user=root
+os_firewall_use_firewalld=true
+openshift_enable_excluders=false
+openshift_install_examples=false
+openshift_clock_enabled=true
+openshift_debug_level="{{ debug_level }}"
+openshift_node_debug_level="{{ node_debug_level | default(debug_level,true) }}"
+osn_storage_plugin_deps=[]
+openshift_master_bootstrap_auto_approve=true
+openshift_master_bootstrap_auto_approver_node_selector={"node-role.kubernetes.io/master":"true"}
+osm_controller_args={"experimental-cluster-signing-duration": ["20m"]}
+osm_default_node_selector="node-role.kubernetes.io/compute=true"
+openshift_enable_service_catalog=false
+
+# Docker
+container_runtime_docker_storage_type=overlay2
+openshift_docker_use_system_container=false
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[masters]
+[nodes]
+[etcd]
+[lb]
+----
+
+. Export the environment variables and run the {product-title} installation playbook:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# export ANSIBLE_ROLES_PATH="/usr/share/ansible/roles/:/usr/share/ansible/openshift-ansible/roles"
+# export ANSIBLE_JINJA2_EXTENSIONS="jinja2.ext.do"
+# ansible-playbook -i /etc/ansible/openshift_3_11.hosts install_ocp.yaml -e @vars.yaml -e @secure_vars.yaml --ask-vault-pass
+----
+
+. Create DNS entries for the routers, for all the infrastructure instances.
+
+. Configure round-robin routing so that the router can pass traffic to applications.
+
+. Create a DNS entry for the {product-title} web console.
+
+. Specify the IP address of the load balancer node.
+
+. Continue to install the cluster following the xref:../install/running_install.adoc#install-running-installation-playbooks[Installing {product-title}] steps, modifying the inventory file according to your cluster's requirements.
```

Comment:
Yes, because now they need to install OpenShift. The previous installation was for RHV.

Reply:
What's now line 263 says ". Export the environment variables and run the {product-title} installation playbook:" If you're not actually running the playbook there, you need to edit that step because it sounds like you're installing OCP twice.